src/main/java/org/betonquest/betonquest/compatibility/citizens/CitizensConversationStarter.java

```diff
@@ -33,6 +33,7 @@
 /**
  * Starts new conversations with Citizen NPCs.
  */
+@SuppressWarnings("PMD.CouplingBetweenObjects")
```

Comment:
I am confused by this div, as this class has no new injections or some thing other new. Can you check that this is actually needed